docs/csharp/language-reference/operators/bitwise-and-shift-operators.md

```diff
@@ -86,7 +86,7 @@ The high-order empty bit positions are set based on the type of the left-hand op
 
 For information about how the right-hand operand of the `>>` operator defines the shift count, see the [Shift count of the shift operators](#shift-count-of-the-shift-operators) section.
 
-## Logical AND operator &amp;
+## <a name="logical-and-operator-"></a> Logical AND operator &amp;
```

Comment:
I'm wondering the addition of the anchor here. The headers have an anchor on the live docs by default. I checked the header id and it is exactly the same: `logical-and-operator-`.

Reply:
This was requested as a work-around by the localization team. It appears that the anchor was different in some translations.

Reply:
@BillWagner, Thanks for clarification. But I'm unable to reproduce it.
I've checked those localized version and the anchor seems to be working without this change.

https://docs.microsoft.com/fr-fr/dotnet/csharp/language-reference/operators/bitwise-and-shift-operators#logical-and-operator-

https://docs.microsoft.com/tr-tr/dotnet/csharp/language-reference/operators/bitwise-and-shift-operators#logical-and-operator-

Reply:
@Youssef1313 I guess those links work because the change introduced in this PR to the *master* is **already** at the *live* branch:
https://raw.githubusercontent.com/dotnet/docs/live/docs/csharp/language-reference/operators/bitwise-and-shift-operators.md

So, I believe, the purpose of this PR is to sync the master branch with the live branch. I don't know how come the live branch got that different from the master, though.

Reply:
@pkulikov , Oh, I didn't think of this. However, I don't think the anchor in the markdown affects the side-bar. And the side-bar shows the correct id. It's confusing.

Reply:
@Youssef1313 This is just for if someone is going to link directly to the heading. Some automated translations were forcing different H2 links on the pages, so if someone linked to what they thought was the default, it would break on some languages. This just creates a specific anchor that we know *won't* be translated and therefor can be reliably linked to.

Reply:
The PR got merged directly to live before by mistake: https://github.com/dotnet/docs/pull/13013

So I just cherry picked to bring it to master